src/commands/wizard/lwcGenerationCommand.ts

```diff
@@ -8,6 +18,70 @@ export class LwcGenerationCommand {
         this.extensionUri = extensionUri;
     }
 
+    static readFileAsJsonObject(
```

Comment:
We are already pulling in `lwc-dev-mobile-core` into this repo and so you can just use `CommonUtils.loadJsonFromFile` to get the job done, which also means that you don't need to add test coverage for this.